x/participationrewards/types/expected_keepers.go

```diff
@@ -22,8 +22,6 @@ type BankKeeper interface {
 	SendCoinsFromModuleToAccount(ctx sdk.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins) error
 	SendCoinsFromModuleToModule(ctx sdk.Context, senderModule, recipientModule string, amt sdk.Coins) error
 	MintCoins(ctx sdk.Context, name string, amt sdk.Coins) error
-	BurnCoins(ctx sdk.Context, name string, amt sdk.Coins) error
```

Comment:
I am mostly sure this is not related to v1.1.0 but any reason?

Reply:
This was removed from expected keepers, because it was only used during dev (before all elements of PR was completed), and I just removed the permissions on PR module account to burn coins; it is no longer used, or usable, in the PR module.